exam001/Views/ViewController.swift

```diff
@@ -0,0 +1,482 @@
+            fourthTextField.heightAnchor.constraint(equalToConstant: 50)
+            ])
+    }
+    
+    private func setupTopBtnConstraints() {
+        topButton.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            topButton.rightAnchor.constraint(equalTo: stack.rightAnchor),
+            topButton.leftAnchor.constraint(equalTo: stack.leftAnchor),
+            topButton.topAnchor.constraint(equalTo: subStack.bottomAnchor, constant: 15),
+            topButton.heightAnchor.constraint(equalToConstant: 50)
+            ])
+    }
+    
+    private func setupBottomBtnConstraint() {
+        bottomButton.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            bottomButton.rightAnchor.constraint(equalTo: stack.rightAnchor),
+            bottomButton.leftAnchor.constraint(equalTo: stack.leftAnchor),
+            bottomButton.topAnchor.constraint(equalTo: topButton.bottomAnchor, constant: 15),
+            bottomButton.heightAnchor.constraint(equalToConstant: 50)
+            ])
+    }
+    
+    private func setupSubmitBtnConstraint() {
+        submitButton.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            submitButton.rightAnchor.constraint(equalTo: stack.rightAnchor),
+            submitButton.leftAnchor.constraint(equalTo: stack.leftAnchor),
+            submitButton.topAnchor.constraint(equalTo: subStack.bottomAnchor, constant: 15),
+            submitButton.heightAnchor.constraint(equalToConstant: 50)
+            ])
+    }
+    
+    private func setupTableViewConstraints() {
+        table.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            table.rightAnchor.constraint(equalTo: stack.rightAnchor),
+            table.leftAnchor.constraint(equalTo: stack.leftAnchor),
+            table.topAnchor.constraint(equalTo: submitButton.bottomAnchor, constant: 15),
+//            table.heightAnchor.constraint(equalToConstant: 500)
+            table.heightAnchor.constraint(greaterThanOrEqualToConstant: 400),
+            table.bottomAnchor.constraint(equalTo: stack.bottomAnchor, constant: -30)
+            ])
+    }
+    
+    private func setupFillerViewConstraints() {
+        fillerView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            fillerView.rightAnchor.constraint(equalTo: stack.rightAnchor),
+            fillerView.leftAnchor.constraint(equalTo: stack.leftAnchor),
+            fillerView.topAnchor.constraint(equalTo: table.bottomAnchor, constant: 15),
+            fillerView.bottomAnchor.constraint(equalTo: stack.bottomAnchor)
+            ])
+    }
+    
+    private func setupStackConstraints() {
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            stack.leadingAnchor.constraint(equalTo: containerView.leadingAnchor, constant: 20),
```

Comment:
int, float, double values included within the class, can be identified within an enum so it will be more readable
Example
```
enum Config {
        static let stackViewLeftPadding = CGFloat(20.0)
}
```

`constant: Config.stackViewLeftPadding`